core/src/main/java/org/apache/accumulo/core/client/admin/NewTableConfiguration.java

```diff
@@ -121,8 +128,22 @@ public NewTableConfiguration setProperties(Map<String,String> prop) {
    */
   public NewTableConfiguration enableSampling(SamplerConfiguration samplerConfiguration) {
     requireNonNull(samplerConfiguration);
-    SamplerConfigurationImpl.checkDisjoint(properties, samplerConfiguration);
-    this.samplerConfiguration = samplerConfiguration;
+    Map<String,String> tmp = new SamplerConfigurationImpl(samplerConfiguration).toTablePropertiesMap();
+    checkDisjoint(properties, tmp, summarizerProps);
+    this.samplerProps = tmp;
```

Comment:
Checking summarizerProps is completely unnecessary here. The function is used in three places.  Only in one of those three places does it actually need to check both.